src/components/Button.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Button(props) {
+  const { name } = props;
+  return (
+    <button type="submit" className="button">
+      {name}
+    </button>
+  );
+}
+
+Button.propTypes = {
+  name: PropTypes.string,
+};
+
+Button.defaultProps = {
+  name: '0',
+};
```

Comment:
Is there any particular reason why you defined a default value for the name props as `'0'` :thinking: ? I guess it makes more sense to define the `name` prop as required since you are always informing its value on the ButtonPanel component.

Reply:
Noted and thank you 😄